.github/workflows/llvm.yml

```
@@ -0,0 +1,55 @@
+name: LLVM Test
```

ditto: that's what other workflows are using

```suggestion
name: LLVM CI
```